### ISSUE TITLE
Loop audio/video setting for embends

### ISSUE THREAD

**Issue by quardbreak** [mastodon/mastodon#20229]
### Pitch

My pitch would go like this: I think explanation for this idea will be simply - repeat feature is a feature for media players. It allow to play media in loop mode, over and over, until closed or stopped manually by user.

Some design ideas how it work in my brain:
- Repeating can be activated by author during file attachment in it's settings. After publishing, this media will play in loop mode when user click play button.
- Users can flip looping in media controls for each playable embed, so can be played even if authors forgot to set it by default, or if user desire to let it play once.

### Motivation

This saves tens of clicks! People love listen music in internet, but clicking gets a bit routine if need to play it a few times more.

In authors side: more ability to present people their ideas - music tracks composed in loop style, videos made with illusion of infinity length. This everything can be more presentable with loop support.

In users side: Pretty sure people prefer to loop music just because content length can be too short to satisfy. If someone want to hear something a couple of times more, then they indeed would love to put it on repeat!

**Comment by KolyaKorruptis** [comment]
I support this request, because it's really not much fun creating a perfect audio loop and then no one can listen to it as it's meant to be. In my opinion it would be fine if there was an optional loop / repeat button on the player that allows the visitor to decide.

**Comment by ivanreese** [comment]
Yes! As a musician, I'm often sharing little perfectly loop-able clips. I'd love nothing more than the ability to set the <audio> or <video> `loop` attribute.
